components/style/properties/properties.mako.rs

```diff
@@ -174,7 +174,7 @@ pub mod shorthands {
         for p in data.longhands:
             if p.name in ['direction', 'unicode-bidi']:
                 continue;
-            if p.internal:
+            if not p.enabled_in_content():
```

Comment:
I don't think this is a huge issue, and otherwise we need to check the enabled-ness individually for each of them after parsing `all`.

I think it's fine for this to include properties via pref, given `all` just parses CSS wide keywords.

This means that in the serialization of a declaration block it may be confusing to see a property which is disabled via pref, but you can't set it to anything else anyway.

Reply:
Oh, sorry, I misread this, nvm.